### PR TITLE
server changes for sandcat watchdog timer

### PR DESCRIPTION
**Pull request by mrengstrom**
necessary changes for the new sandcat profile addition of 'watchdog', and a small change so future additions are ignored and don't cause an exception.

### CODE REVIEW

**Comment by privateducky** [comment]
playing devil's advocate here.. could we utilize sleep instead of adding a new parameter? for example, if sleep = 0/0, then the agent will auto-die the next time it can't make a successful beacon.

**Comment by mrengstrom** [comment]
well the 0/0 doesnt go back to the agent, as it is turned into a random value between the limits by jitter in the BaseWorld class.  So it would have to be on the agent side that if the value passed back for sleep is 0 (or we could do a negative) then to kill the agent.  But I could see that being feasible and already hooked into the UI.

File: app/objects/c_agent.py [hunk 1]
```diff
@@ -45,6 +46,7 @@ def __init__(self, paw, host=None, username=None, architecture=None, platform=No
         self.privilege = privilege
         self.c2 = c2
         self.exe_name = exe_name
+        self.watchdog = watchdog
```
Comment (privateducky) [inline app/objects/c_agent.py]: let's make it an int here - and remove the conversion in rest_api.py

**Comment by privateducky** [comment]
let's update the Docs - sandcat page to describe this new option

**Pull request merged**